cli/command_analytics.go

```diff
@@ -0,0 +1,68 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/bitrise-io/bitrise/analytics"
+	"github.com/urfave/cli"
+)
+
+var globalTracker analytics.Tracker
+
+func logPluginCommandParameters(name string, _ []string) {
+	// Plugin command parameters are routed into the function but are not processed yet because it is complex to correctly
+	// parse the arguments without knowing the structure. If we notice that our users do use plugins, then we can add
+	// plugin specific argument parsers.
+	sendCommandInfo(fmt.Sprintf(":%s", name), "", []string{})
+}
+
+func logSubcommandParameters(c *cli.Context) {
```

Comment:
The context for the command and subcommand are almost identical without any explicit value indicating if this is a command or subcommand execution. The only difference I have found is that commands have a single empty parent context while subcommands have two empty in their parent chain. I thought about using it but it felt like this is an implementation detail that too implicit and is not guaranteed to stay like this. 

So in the end I chose to have a separate function instead because at the call site we can clearly indicate if it is a command or subcommand.

Reply:
Updated the PR based on our discussion.